### ISSUE TITLE
Add aXe-core tests

### ISSUE THREAD

**Issue by toolness** [18F/django-uswds-forms#8]
We should add a selenium test that just visits the example app home page, and runs [aXe](https://www.deque.com/products/axe/) on its following states:

* As the result of a GET request;
* As the result of a POST request with an invalid form;
* As the result of a POST request with a valid form.

**Comment by toolness** [comment]
Er, now that the "example app" is really an "example gallery", we should just crawl every example page. We probably don't need to actually submit forms because we have separate example pages showing errors, I think.

**Comment by toolness** [comment]
As of today, I've manually visited all of the examples and ensured there are no violations (well, there was one, which I fixed in b0abc467b46c70970d8530c8eacec881bd58599d).

Integrating aXe-core with our automated test suite will still certainly be useful in the long-term, however. I just loathe dealing with Selenium so I am putting it off for now. (We could just create a PhantomJS script that visits the pages, I suppose, to bypass Selenium entirely, since there's very little Selenium actually needs to do here.)
